credentials/apps/badges/models.py

```diff
@@ -380,6 +382,30 @@ class CredlyBadge(UserCredential):
         default=STATES.created,
     )
 
+    def as_badge_data(self, badge_template: CredlyBadgeTemplate) -> BadgeData:
```

Comment:
We don't need `badge_template` here since UserCredential has a link to its credential.

Reply:
Also, pls use it in 
- notify_badge_awarded
- notify_badge_revoked